kayenta-core/kayenta-core.gradle

```diff
@@ -2,6 +2,7 @@ dependencies {
   api "redis.clients:jedis"
   api "com.netflix.spinnaker.kork:kork-core"
   api "com.netflix.spinnaker.kork:kork-web"
+  api "com.netflix.spinnaker.kork:kork-plugins"
```

Comment:
Think this should be `implementation` under `kayenta-web` until we write an ExtensionPoint for kayenta.